models/src/main/kotlin/motif/models/Dependencies.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2018 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package motif.models
+
+import motif.ast.IrClass
+import motif.ast.IrMethod
+
+/**
+ * [Wiki](https://github.com/uber/motif/wiki#dependencies)
+ */
+class Dependencies(val clazz: IrClass, val scope: Scope) {
```

Comment:
is `clazz` the right kotlinism? @tyvsmith

Reply:
Good question - I could use backticks. Makes the field reference a bit ugly but I can go either way.

Reply:
How about something descriptive like dependencyClass.

Reply:
As discussed offline - since IrClass is the primitive representation of a class AST and this is a general model used for multiple difference reasons (graph processing + codegen), it's appropriate to leave the field name unqualified.